examples/geometry/tests/test_generated_bindings.rs

```diff
@@ -1,5 +1,5 @@
 uniffi_macros::build_foreign_language_testcases!(
-    "src/geometry.udl",
+    "src/lib.rs",
```

Comment:
> I think it might be better if we passed a reference to the crate itself rather than to a specific file,

Months later, I still have a pretty strong gut feeling that treating the crate as the unit of UniFFIcation is the right way to go.